doc/reference/actions.rst

```diff
@@ -62,13 +62,14 @@ The following constructions are available:
 - ``(bash <cmd>)`` to execute a command using ``/bin/bash``. This is obviously
   not very portable.
 - ``(diff <file1> <file2>)`` is similar to ``(run diff <file1> <file2>)`` but
-  is better and allows promotion. See :doc:`promotion` for more details.
+  is better and allows promotion. See :doc:`../concepts/promotion` for more
+  details.
 - ``(diff? <file1> <file2>)`` is similar to ``(diff <file1>
   <file2>)`` except that ``<file2>`` should be produced by a part of the
   same action rather than be a dependency, is optional and will
```

Comment:
```suggestion
  same action rather than be a dependency. It's optional and will
```
I'm not sure this should be a separate sentence like this or if this is what was meant, but the sentence as-is doesn't make sense. The part up until "than be a dependency" does, but what is optional and be consumed by `diff`?

Reply:
file2 is optional and is consumed by `diff?`. I agree that the sentence is not great at the moment. I'll merge the move and will try to improve that.